[7.9.0] error: constructor DynamicFontModule in class DynamicFontModule cannot be applied to given types;       new DynamicFontModule()

<!--
NOTE: please submit only bug reports here, any new questions or feature requests should be submitted in Discussions:
https://github.com/wix/react-native-ui-lib/discussions
 -->

## Description

<!--
A clear and concise description of what is the bug.
-->

After updating to `7.9.0` Android build throws this error: 

```
> Task :app:compileDebugJavaWithJavac FAILED
xxxxx/android/app/build/generated/rncli/src/main/java/com/facebook/react/PackageList.java:222: error: constructor DynamicFontModule in class DynamicFontModule cannot be applied to given types;
      new DynamicFontModule(),
      ^
  required: ReactApplicationContext
  found:    no arguments
  reason: actual and formal argument lists differ in length
Note: xxxxx/android/app/src/main/java/com/hiki/MainApplication.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
1 error
```

offending PR: https://github.com/wix/react-native-ui-lib/pull/2736

![Screenshot 2023-10-12 at 11 08 20](https://github.com/wix/react-native-ui-lib/assets/717975/8522496c-3713-4da5-a95b-6e45c3f1c325)



### Related to

- [x] Components
- [ ] Demo
- [ ] Docs
- [ ] Typings

### Steps to reproduce

<!--
Steps to reproduce the behavior:
1. update/install 7.9.0
2. Build on Android
-->

### Environment

<!--
Fill in your RNUILib and React Native versions below.

List other libraries if relevant.
-->

- React Native version: 0.72.5 Paper
- React Native UI Lib version: 7.9.0

### Affected platforms

- [x] Android
- [ ] iOS
- [ ] Web

--- a/react-native.config.js
+++ b/react-native.config.js
@@ -8,11 +8,11 @@ module.exports = {
       // },
       android: {
         sourceDir: './lib/android/',
-        packageImportPath: `import com.wix.reactnativeuilib.dynamicfont.DynamicFontModule;
+        packageImportPath: `import com.wix.reactnativeuilib.dynamicfont.DynamicFontPackage;
 import com.wix.reactnativeuilib.highlighterview.HighlighterViewPackage;
 import com.wix.reactnativeuilib.keyboardinput.KeyboardInputPackage;
 import com.wix.reactnativeuilib.textinput.TextInputDelKeyHandlerPackage;`,
-        packageInstance: `new DynamicFontModule(),
+        packageInstance: `new DynamicFontPackage(),
       new HighlighterViewPackage(),
       new TextInputDelKeyHandlerPackage(),
       new KeyboardInputPackage(getApplication())`

